### PR TITLE
speed dependant on length

### LINKED ISSUE #360: Adjust messages speed
See also https://github.com/PerfectFit-project/testing-tickets/issues/3

Now the speed of messages is fixed [in the broker](https://github.com/PerfectFit-project/niceday-components/blob/f42b5d6365db758bd82a099a1ba8a5b16148f8c4/niceday-broker/index.js#L79), and they are delivered with 3s of delay one after the other. 

- [x] Adjust delay to the length of the previous message
- [x] Add delay also in the custom_channels

### PR DESCRIPTION
**Pull request by wbaccinelli**
Fixes: https://github.com/PerfectFit-project/virtual-coach-issues/issues/360

The delay in the broker has been changed to be adapted to the previous message length.

### CODE REVIEW

File: niceday-broker/index.js [hunk 1]
```diff
@@ -73,22 +77,28 @@ function sleep(ms) {
...
+        attachmentIds: message.metadata || [],
       };
-      if ('metadata' in message) {
-        attachment.attachmentIds = message.metadata;
-      }
+
       sendMessage(message.text, parseInt(message.recipient_id, 10), attachment);
-    });
+
+      if (ENVIRONMENT === 'prod') {
```
Comment (raar1) [inline niceday-broker/index.js:92]: So we're not testing this in dev mode currently? Or is all testing happening through prod?

**Review by raar1 — APPROVED** [review]
The delay (set by words per second) is implemented here as well as in the Rasa actions (I think, from another PR). Why does it need to be implemented in both places?

File: niceday-broker/index.js (hunk 1 above)
Comment (wbaccinelli) [reply niceday-broker/index.js:92]: The local testing is in dev mode, but the deployed version is in prod mode, to resemble the actual behavior

**Comment by wbaccinelli** [comment]
> The delay (set by words per second) is implemented here as well as in the Rasa actions (I think, from another PR). Why does it need to be implemented in both places?

Because our infrastructure is way too complicated, so there are two ways the messages are exchanged with NiceDay:
- The broker, which intervenes when the messages are generated during the conversation, in the Rasa stories
- The custom_channels of Rasa, that we need for running the external triggers and that uses the niceday_api.  

So we need to change this stuff in both the places

**Pull request merged**